components/slider/script.js

```diff
@@ -123,24 +123,29 @@ class Slider extends BaseComponent {
      * @param {HTMLElement} element
      * @returns {number} - the size in pixels.
     */
-    _getPxSizeWithoutUnits(element) {
-        let unitsName = this.units.sizePX;
-        if (!navigator.userAgent.includes('Cohtml')) unitsName = unitsName.substring(0, unitsName.length - 2);
-
-        const size = getComputedStyle(element)[unitsName];
-        return Number(size.substring(0, size.length - 2));
+    getBoundingClientSize(element) {
```

Comment:
Maybe change the name as it's very similar to `getBoundingClientRect` and it's confusing.